docs/updating/6-0.md

```diff
@@ -335,59 +341,62 @@ Ensure `null` is not passed in as a value to the `placeholder` property. We reco
 `ion-modal` now uses the Shadow DOM. Update any styles targeting the internals of `ion-modal` to use either the [ion-modal CSS Variables](../api/modal#css-custom-properties) or the [ion-modal CSS Shadow Parts](../api/modal#css-shadow-parts):
 
 **Before**
+
 ```css
 ion-modal .modal-wrapper {
-  ...
+  ...;
```

Comment:
These errant semicolons make me wonder if it's possible to exclude code snippets within md files 🤔 If not, maybe we should exclude md files altogether?

Reply:
I spoke with Sean about this after I noticed it and we decided to just make sure that it was valid css. I changed it to code comments instead to make prettier happy: https://github.com/ionic-team/ionic-docs/pull/2948/commits/21c343268968834514590ea2d0854a19d71e79b9